scripts/irods/test/test_control_plane.py

```diff
@@ -84,6 +84,27 @@ def test_shutdown_local_server(self):
             finally:
                 IrodsController().start()
 
+    def test_reload_command(self):
```

Comment:
is this test... identical to the one in test_configuration_reload.py?

Reply:
Not exactly, it tests fewer things, so it should probably just live over there now